api/v1/components/drivemgr.go

```diff
@@ -18,9 +18,6 @@ package components
 
 // DriveMgr represents drive manager node component
 type DriveMgr struct {
-	Image            *Image `json:"image,omitempty"`
-	Endpoint         string `json:"string"`
-	DeployConfig     bool   `json:"deployConfig"`
-	AmountOfLoopback uint64 `json:"amountOfLoopback"`
```

Comment:
Why these settings are removed?

Reply:
These setting are configured throw ConfigMap, but it's created separately using `kubectl apply`. To deploy drivemgr we need to know only if we should use config or not. We don't really need these values for deployment.

Reply:
ConfigMap is deployed via helm and those parameters allow to override default values.